index.js

```diff
@@ -25,62 +25,61 @@ var copy = require('copy-to');
  *   - {Object} extendTypes
  */
 
-module.exports = function (opts) {
+module.exports = function(opts) {
   opts = opts || {};
-  var detectJSON = opts.detectJSON;
-  var onerror = opts.onerror;
+  const {detectJSON} = opts;
+  const {onerror} = opts;
 
-  var enableTypes = opts.enableTypes || ['json', 'form'];
-  var enableForm = checkEnable(enableTypes, 'form');
-  var enableJson = checkEnable(enableTypes, 'json');
-  var enableText = checkEnable(enableTypes, 'text');
-  var enableXml = checkEnable(enableTypes, 'xml');
+  const enableTypes = opts.enableTypes || ['json', 'form'];
+  const enableForm = checkEnable(enableTypes, 'form');
+  const enableJson = checkEnable(enableTypes, 'json');
+  const enableText = checkEnable(enableTypes, 'text');
+  const enableXml = checkEnable(enableTypes, 'xml');
+  const enableHtml = checkEnable(enableTypes, 'html');
```

Comment:
hmmm, I think there is no need to add a `html` type here. we can get text/html by:

```
bodyparser({
  extendTypes: {
    text: [ 'text/plain', 'text/html' ],
  }
})
```

Reply:
@3imed-jaberi can you revert this `enableHtml` feature here? :)

Reply:
I'v reverted `enableHtml` option, we can use text parser to parse html content.:)

https://github.com/koajs/bodyparser/commit/4d931c634e9b59a843152f56d68b3ef2e1719675#diff-faf36523e15f6c0b9ee07674a066b3ecR262

Reply:
@dead-horse, 
sorry bro, I did not see your comment 😟 ..